demux/cli/samplesheet.py

```diff
@@ -148,6 +156,16 @@ def get_project(project):
                                    f"only!", fg='red'))
             context.abort()
 
+        if pad and not indexlength:
+            click.echo(click.style(f"Please specify an index length when using the pad option!"
+                                   f"Use --longest or --indexlength. Nota that padding only works"
+                                   f"in combination with d10 indxes!", fg='red'))
+            context.abort()
+
+        if pad and indexlength != 20:
+            click.echo(click.style(f"Padding is only allowed in combination with d10 indexes!", fg='red'))
```

Comment:
I know what d10 is considering the context and not everyone might know that. Can we either name it 10+10 or d10 consistently. 10+10 seems more descriptive, what do you think?

Reply:
Good suggestion, but I will make it 'dual 10' instead, according to the end user representative of the production team, that's the most commonly used terminology.